include/zephyr/dt-bindings/clock/mspm0_defines.h

```diff
@@ -25,4 +25,27 @@
 #define RATE_7 6
 #define RATE_8 7
 
+// check DL_TIMER_PWM_MODE
+#define PWM_EDGE_ALIGN    0
+#define PWM_EDGE_ALIGN_UP 32
+#define PWM_CENTER_ALIGN  16
```

Comment:
I guess these numbers come from the spec?

Reply:
These values are extracted directly from TI's hal for `timer.h`